planet.ini

```diff
@@ -219,3 +219,5 @@ name = Shivam Singhal (championshuttler)
 [https://prabhuss.wordpress.com/feed/]
 name = Prabhu Sharan Singh (prabhuss)
 
+[https://pradhvanbisht.wordpress.com/feed/]
+name = Pradhvan Bisht (pradhvan)
```

Comment:
Can you please add new line at the end of file?

Reply:
@bhavin192 done